StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkURLGenerator.swift

```diff
@@ -4,36 +4,107 @@
 //
 
 import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
 
 struct LinkURLParams: Encodable {
     struct MerchantInfo: Encodable {
-        let businessName: String
-        let country: String
+        var businessName: String
+        var country: String
     }
     struct CustomerInfo: Encodable {
-        let country: String
-        let email: String
+        var country: String
+        var email: String?
     }
     struct PaymentInfo: Encodable {
-        let currency: String
-        let amount: Int
-    }
-    let path = "mobile_pay"
-    let integrationType = "mobile"
-    let publishableKey: String
-    let merchantInfo: MerchantInfo
-    let customerInfo: CustomerInfo
-    let paymentInfo: PaymentInfo
-    let returnUrl: URL
-    let experiments: [String]
-    let flags: [String]
-    let loggerMetadata: [String]
-    let locale = Locale.current.toLanguageTag()
+        var currency: String
+        var amount: Int
+    }
+    enum PaymentObjectMode: String, Encodable {
+        case link_payment_method
+        case card_payment_method
+    }
+    var path = "mobile_pay"
+    var integrationType = "mobile"
+    var paymentObject: PaymentObjectMode
```

Comment:
Just confirming -- in addition to changing everything from var to let we added:
* paymentObject
* stripeAccount
* paymentUserAgent

Reply:
Yep. Still TBD, we're still working out the exact shape of this request.